Minio.Examples/Cases/DisableVersioning.cs

```diff
@@ -0,0 +1,42 @@
+/*
+ * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2017 MinIO, Inc.
```

Comment:
```suggestion
 * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
```

Reply:
please make sure that this change is done on all the files that were newly added

Reply:
Change done.